node/primitives/src/approval.rs

```diff
@@ -77,6 +77,38 @@ pub enum AssignmentCertKind {
 	},
 }
 
+/// Certificate is changed compared to `AssignmentCertKind`:
+/// - introduced RelayVRFModuloCompact
+#[derive(Debug, Clone, Encode, Decode, PartialEq, Eq)]
+pub enum AssignmentCertKindV2 {
+	/// An assignment story based on the VRF that authorized the relay-chain block where the
+	/// candidate was included combined with a sample number.
+	///
+	/// The context used to produce bytes is [`RELAY_VRF_MODULO_CONTEXT`]
+	RelayVRFModulo {
+		/// The sample number used in this cert.
+		sample: u32,
+	},
+	/// Multiple assignment stories based on the VRF that authorized the relay-chain block where the
+	/// candidate was included combined with a sample number.
+	///
+	/// The context used to produce bytes is [`RELAY_VRF_MODULO_CONTEXT`]
+	RelayVRFModuloCompact {
+		/// The number of samples.
+		sample: u32,
```

Comment:
This should go away, and just read from the config: `relay_vrf_modulo_samples`.

Reply:
Ahh okay I read the wrong part above then I guess.